iam-keycloak-migration.sh

```diff
@@ -557,8 +557,20 @@ then
   # Get the name of the keycloak client
   cp4iKeycloakClientId="$(oc get integrationkeycloakclient.keycloak.integration.ibm.com -l app.kubernetes.io/name=ibm-integration-platform-navigator -n "$servicesNamespace" -o jsonpath='{.items[0].spec.client.clientId}')"
 
-  keycloakAdminSecretName="cs-keycloak-initial-admin"
-  keycloakAdminPass="$(oc get secret "$keycloakAdminSecretName" -n "$servicesNamespace" -o jsonpath="{.data.password}" | base64 -d)"
+  keycloakIntegrationUserName="internal-keycloak-user"
+  keycloakIntegrationAdminSecretName="internal-keycloak-user-secret"
+  
+  # Default to the CS values, until we can validate that our internal IKU is fully reconciled
+  secretToFetch="cs-keycloak-initial-admin"
+  keycloakAdminUser="admin"
+  
+  # Check the integration internal IKU is reconciled and if so, use t instead of the CS one
+  integrationIKUPhase="$(oc get integrationkeycloakuser "$keycloakIntegrationUserName" -n "$servicesNamespace" -o jsonpath='{.status.phase}')"
```

Comment:
Are we sure this will work if the client is on a previous version which doesn't have the `internal-keycloak-user` IKU?
I think this will throw an error on screen which we probably don't want to the end user to see.

We probably want to add to silently error -
```suggestion
  integrationIKUPhase="$(oc get integrationkeycloakuser "$keycloakIntegrationUserName" -n "$servicesNamespace" -o jsonpath='{.status.phase}' --ignore-not-found=true)"
```

Reply:
@tom-youd I don't think that's needed as the command won't return anything if the IKU doesn't exist:

Reply:
apologies, you're correct, I just realised I ran it with your version. I'll add it in!
<img width="742" alt="image" src="https://github.com/user-attachments/assets/4104051d-9e7c-4d57-adb8-b3669e05534f" />